staging/src/k8s.io/client-go/util/workqueue/doc.go

```diff
@@ -22,5 +22,6 @@ limitations under the License.
 //     will only be processed once.
 //   - Multiple consumers and producers. In particular, it is allowed for an
 //     item to be reenqueued while it is being processed.
+//     In this case it will be processed again.
```

Comment:
I suppose this was implied by saying "reenqueued" instead of "added" and it is what one would expect, but I still wasn't 100% sure when reading it.